draft-ietf-quic-transport.md

```diff
@@ -1224,11 +1227,11 @@ be unable to continue using the connection IDs with the active connection.
 
 An endpoint SHOULD limit the number of connection IDs it has retired locally and
 have not yet been acknowledged. An endpoint SHOULD allow for sending and
-tracking a number of RETIRE_CONNECTION_ID frames of at least twice the
-active_connection_id limit. An endpoint MUST NOT forget a connection ID without
-retiring it, though it MAY choose to treat having connection IDs in need of
-retirement that exceed this limit as a connection error of type
-CONNECTION_ID_LIMIT_ERROR.
+tracking a number of RETIRE_CONNECTION_ID frames of at least twice the value of
+the active_connection_id_limit transport parameter. An endpoint MUST NOT forget
```

Comment:
How about: "The number of RETIRE_CONNECTION_ID frames that an endpoint is allowed to send and track SHOULD be at least twice the value of the active_connection_id_limit transport parameter"

Reply:
This isn't about giving the endpoint (the one that sends RETIRE_CONNECTION_ID) a limit.  Saying "is allowed to send and track" implies that the endpoint cannot send more than this.  The point is that it might need to apply its own limits to avoid having to track too many outstanding frames.

Reply:
Reading it now makes this more reasonable ... I think I must have been reading it differently earlier. I don't mind dropping this one ... but I'd like to discuss the one above.